tests/unit/classifier/svm/LibLinear_unittest.cc

```diff
@@ -1,11 +1,11 @@
 /*
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 3 of the License, or
- * (at your option) any later version.
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
  *
- * Written (W) 2014 pl8787
+ * Authors:
+ *	Written (W) 2014 pl8787
+ *	Refactored (R) 2018 Elfarouk Yasser
```

Comment:
just add yourself as author in the standard way...makes later processing of headers easier

Reply:
Fixed